src/features/smartSearch/components/inputs/StyledSelect.tsx

```diff
@@ -10,6 +10,8 @@ const useStyles = makeStyles((theme) => ({
   MuiSelect: {
     fontSize: theme.typography.h4.fontSize,
     padding: 0,
+    maxWidth: '700px',
```

Comment:
I think something like 3-400px should be enough. Or, another option would be to use the `truncateOnMiddle()` utility function from `src/utils/stringUtils` module, and truncate to something like 40 characters, which will make sure both beginning and end are included.